monitorrent/plugins/clients/transmission.py

```diff
@@ -110,25 +115,40 @@ def add_torrent(self, torrent, torrent_settings):
         client = self.check_connection()
         if not client:
             return False
-        try:
-            torrent_settings_dict = {}
-            if torrent_settings is not None:
-                if torrent_settings.download_dir is not None:
-                    torrent_settings_dict['download_dir'] = torrent_settings.download_dir
-
-            client.add_torrent(base64.b64encode(torrent).decode('utf-8'), **torrent_settings_dict)
-            return True
-        except transmissionrpc.TransmissionError:
-            return False
+        torrent_settings_dict = {}
+        if torrent_settings is not None:
+            if torrent_settings.download_dir is not None:
+                torrent_settings_dict['download_dir'] = torrent_settings.download_dir
+        client.add_torrent(base64.b64encode(torrent).decode('utf-8'), **torrent_settings_dict)
+        return True
```

Comment:
I think we can remove such `return True` because right now if there are no exception, than it is ok :)